### PR TITLE
Complete Configu CLI - Login command

### PR DESCRIPTION
**Pull request by RichardAkman**
Partially completes #503 by implementing login command

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -36,11 +36,15 @@
     "start": "pnpm configu"
   },
   "dependencies": {
-    "@configu/sdk": "workspace:*",
     "@configu/common": "workspace:*",
+    "@configu/sdk": "workspace:*",
+    "axios": "^1.7.4",
     "clipanion": "^4.0.0-rc.4",
     "consola": "^3.2.3",
+    "inquirer": "12.0.0",
```
Comment (davidmeirlevy) [inline packages/cli/package.json:44]: consider to use the next version:
https://www.npmjs.com/package/@inquirer/prompts

**Review by davidmeirlevy — COMMENTED** [review]
it's better to use the next version of inquirer, but regardless - it's fine.

File: packages/cli/package.json (hunk 1 above)
Comment (RichardAkman) [reply packages/cli/package.json:44]: Will open an issue for this, it's not necessary for this migration PR.

**Review by davidmeirlevy — APPROVED** [review]

File: packages/cli/package.json (hunk 1 above)
Comment (RichardAkman) [reply packages/cli/package.json:44]: Follow up issue: https://github.com/configu/configu/issues/579

**Pull request merged**